IliasDownloaderUniMA/__init__.py

```diff
@@ -198,8 +289,19 @@ def scanFolder(self, course_name, url_to_scan):
 			print(f"Scanning Folder...\n{file_path}\n{url}")
 			print("-------------------------------------------------")
 		for v in videos:
-			el_url = urljoin(self.base_url, v.find('source')['src'])
-			el_name = v.find('div', {'class': 'ilc_media_caption_MediaCaption'}).get_text()
+			v_url = v.find('source')['src']
```

Comment:
This breaks if the media inside `ilc_media_caption_MediaCaption` isn't a video. Example:

<img width="573" alt="Bildschirmfoto 2020-10-23 um 12 19 07" src="https://user-images.githubusercontent.com/8471326/96993168-37455600-152b-11eb-9803-99a974bbd372.png">

```suggestion
			try:
				v_url = v.find('source')['src']
			except:
				continue
```

Reply:
Didn't think of this problem. Looks like a solid change, I'll commit it.

Sidenote: I think we have to implement or import a bw sync&share downloader at some point. 
Some profs upload some files, only saw videos thus far, there and link them back, instead of uploading them directly :roll_eyes: